bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_batchgenerator.py

```diff
@@ -205,7 +205,7 @@ def __init__(
             chunk_size,
             batch_size,
             self.given_columns,
-            filters,
+            " & ".join(filters),
```

Comment:
There is that problem, yes, but also there need to be parenthesis around the individual filters. Imagine the case where `filters = ["a || b", "c]`.
The `join` would turn this into `a || b && c`, but `&&` is evaluated before `||` so now the logic is wrong!

Since @DanteNiewenhuis has moved on from the project, I will fix this up myself so that we can merge the PR.